### DISCUSSION TITLE
Why there are people I don't follow?

### DISCUSSION THREAD

**Discussion by ris58h** [community/community#13439] [Other Feature Feedback, Questions, & Ideas]
There are `starred a repository` posts from people I don't even know. Why GH? Why?

**Comment by ArwynFr** [comment]
What I found was these people have starred and contributed to repositories I have starred. This means that, if I contribute to a popular repository, hundreds of people that have starred it will see all my activity on other repositories in their timeline.

This is extremely awkward.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by jfullstackdev** [reply]
maybe not the exact request you want, but this is a great update and I think they will be working further for more granularity
